amgut/db/patches/0010.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE ag.external_survey ( 
```

Comment:
I'm not sure type is a good descriptor here, since they are full surveys in and of themselves, with names, not types.